airbyte-cdk/bulk/toolkits/extract-jdbc/src/main/kotlin/io/airbyte/cdk/read/JdbcPartitionReader.kt

```diff
@@ -48,15 +47,10 @@ sealed class JdbcPartitionReader<P : JdbcPartition<*>>(
     private val outData: ObjectNode = Jsons.objectNode()
 
     private val msg =
-        AirbyteMessage()
```

Comment:
Nit: Any way to more strongly discourage the use of this object in the framework?

Reply:
Can you elaborate, please? I'm not sure what you mean.

Reply:
A nit, and it might not matter if this is an internal concept, but if the partitionReaders we make in the toolkit should use `AirbyteRecordMessage` rather than `AirbyteMessage`, is there a way to mark it deprecated or something like that?

Reply:
Ah, I see. No, it's not quite like that. You can use either-or and the outcome is the same! Therefore, since it's the same, let's use the more specific `AirbyteRecordMessage`.